Update - 'duplicate' functionality

rvegele opened the pull request:
A quick way to copy-paste selected elements for those coming from Figma and similar tools where such functionality exists. 

I have a feeling that there's more to creating this pull request (adding something to tests?) but in my build it seems to work nicely :) 

I've also moved the 'Copied' toast from the copySelected() into copy() as it was also showing up while using cut() - 2 toasts (Copied and Cut). I understand that under the hood it copies selection into the clipboard, but visualising that with a toast event when cutting or duplicating is not necessary, as only the end-result is important for the user.

rvegele committed:
Add Duplicate - EditorDocument.tsx

Added Duplicate

rvegele committed:
Duplicate in nodegrapheditor.ts

Added duplicate

mrtamagotchi commented on packages/noodl-editor/src/editor/src/views/nodegrapheditor.ts:
```diff
@@ -1095,6 +1094,25 @@ export class NodeGraphEditor extends View {
+      nodeset: ns,
+      x: this.latestMousePos.x,
+      y: this.latestMousePos.y,
+      toastMessage: 'Duplicate'
```
This should be "Duplicated" to match the past tense of the other action UI copy.

rvegele replied:
Yes, updated!

rvegele committed:
Update: nodegrapheditor.ts 'Duplicate' to 'Duplicated'

Update: nodegrapheditor.ts 'Duplicate' to 'Duplicated' string in duplicate() function
